validation/lagom-validation/matriculation-management.md

```diff
@@ -0,0 +1,16 @@
+# MatriculationData Validation
+Every invalid equivalence class returns an error. Every valid class returns an empty sequence.
+
+## SubjectMatriculation
+
+### fieldOfStudy (String)
+*validFieldsOfStudies* := {"Computer Science", "Philosophy", "Media Sciences", "Economics", "Mathematics", "Physics","Chemistry", "Education", "Sports Science", "Japanology", "Spanish Culture", "Pedagogy", "Business Informatics", "Linguistics"}
+- vFieldOfStudy1: fieldOfStudy ∈ *validFieldsOfStudies*
+- vFieldOfStudy1: fieldOfStudy ∉ *validFieldsOfStudies*
+
+### semesters (Array[String])
+Each semester in the list is validated as follows:  
+Valid semesters are of the form "SS2016" and "WS2017/18"
+- vSemester1: semester is of a valid format for semesters
+- iSemester1: semester is the empty String
+- iSemester2: semester is not the empty String AND semester is not in a valid format for semesters
```

Comment:
Do you check if 2017/18 are consecutive years? If so, you should explicitly state that.